### ISSUE TITLE
Failed to synchronize registry data from server ATM6 - To The Sky 

### ISSUE THREAD

**Issue by Pankekus** [killpowa/Twickt-Launcher#1278]
**Describe the bug**
Can't connect to a locally hosted server on the GDLauncher version of ATM6 - To The Sky, both server and client are 1.2.0 version. There is a connection when using Curseforge, didn't try with MultiMC. Edit: Fixed with copying almost all the files from Curseforge ATM6 - To The Sky to the GDLauncher instance. I can provide logs for that too.

**To Reproduce**
1. Download ATM6 - To The Sky 
2. Run with  https://pastebin.com/v2BSsB1k arguments.
3. Connect to a locally hosted ATM6 - To The Sky server.

**Expected behavior**
Connecting to the server correctly. 

**Possible solutions**
Broken or outdated config files. (Copying and pasting config folder didn't work)

**Screenshots**

**Operating System:**
- OS: Windows 10
- Version: 21H1
- Java Version: Java 8 (build 1.8.0_321-b07) 
- Minecraft Version: 1.16.5
- Mod Loader: Forge

**Additional context**
[debug.log](https://github.com/gorilla-devs/GDLauncher/files/8080356/debug.log) - client
[debug.log](https://github.com/gorilla-devs/GDLauncher/files/8080342/debug.log) - server

**Comment by commandhat** [comment]
Not a contributor, but...

This is a mod error -- registry data usually indicates there's a mod or two missing from the client that's necessary, or that the server has a couple extra mods.

Historyically, this is because for whatever reason GDLauncher fails to download a given mod for the client -- usually because GDlauncher is set to only accept mods in Beta or Release status, while one of the mods in the pack is only at Alpha release, or because the modpack calls for an Alpha version of a mod when there are older Beta versions available.

To check and fix this if it's true, open GDlauncher, and click on the settings cog. There'll be a dropdown marked 'Preferred Curse Release Channel' -- this should be set to Alpha to ensure compatibility. Here's a picture indicates this dropdown if you have trouble finding it in the settings list.

![gdlb](https://user-images.githubusercontent.com/3821018/154328409-9ef78e79-fa72-4132-8610-b33b6830a57a.png)
.
Edit: I should also add that this seems to only be a recent issue -- I suspect someone either accidentally, or on purpose, changed the default of this dropdown from from 'Alpha' to 'Beta' -- which makes sense considering the recent security hullabaloo that went on

**Comment by Pankekus** [comment]
Tried that and the problem is still occurring, only after copying almost all the files from Curseforge and running a repair I'm able to connect.

**Comment by commandhat** [comment]
the debug.log for your client seems to match the one you supplied for the server.

I'll try downloading from CurseForge and then downloading from GDLauncher and diffing the two, give me a minute...

**Comment by Xplodin** [comment]
okay so gdlauncher misses enviromentalenergy mod. 
it fails to download it due to apparent false positive

**Comment by Eskaan** [comment]
Duplicate of #1223
